troposphere/settings/__init__.py

```diff
@@ -1,4 +1,5 @@
+from troposphere.settings.default import *
 try:
-    import troposphere.settings.local
+    from troposphere.settings.local import *
```

Comment:
I think changing from `import foo` to `from foo import *` will change how the imports exist in the namespace, won't it?  Is that the desired effect?  

Also there might be namespace conflicts introduced by using `import *`.